docs/get-started/viam.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Viam in 3 minutes"
-linkTitle: "Viam in 3 minutes"
+title: "A tour of Viam"
+linkTitle: "Tour of Viam"
```

Comment:
What if we call this "What is Viam?" I think we did that back in the day...what was the reason for not doing this? I think it's a frequently asked question and is what many users might be wondering when they land on our docs site (as well as other site). I often find myself wondering "But what actually *is* xyz tech company/product" on different docs sites.